emails/utils.py

```diff
@@ -447,15 +449,29 @@ def count_all_trackers(html_content):
     )
 
 
-def remove_trackers(html_content, level="general"):
+def remove_trackers(html_content, from_address, datetime_now, level="general"):
```

Comment:
Nice job getting into the backend code too! Shipping both together raises my confidence that the two parts will work. 🥞 

I like passing the timestamp into the function, so that all the links in the email will get the same timestamp.